tests/unit/test_computations.py

```diff
@@ -0,0 +1,112 @@
+"""Test the function of computations module."""
+
+import numpy as np
+import pytest
+
+from wristpy.core import computations, models
+from wristpy.io.readers import readers
+
+SIGNAL_LENGTH = 20
+EPOCH_LENGTH = 5
+
+
+def test_moving_mean_epoch_length_is_negative() -> None:
+    """Test error if the epoch length is negative."""
+    tmp_data = np.array([1, 2, 3])
+    time = readers.unix_epoch_time_to_polars_datetime(np.array([1, 2, 3]), "s")
+    tmp_Measurement = models.Measurement(measurements=tmp_data, time=time)
+
+    with pytest.raises(ValueError):
+        computations.moving_mean(tmp_Measurement, epoch_length=-EPOCH_LENGTH)
+
+
+def test_moving_mean_one_column() -> None:
+    """Test the functionality of the moving mean function for 1D Measurement."""
+    test_data = np.arange(0, SIGNAL_LENGTH)
+    test_time = readers.unix_epoch_time_to_polars_datetime(
+        np.arange(0, SIGNAL_LENGTH), "s"
+    )
+    expected_mean = np.array([2.0, 7.0, 12.0, 17.0])
+    test_measurement = models.Measurement(measurements=test_data, time=test_time)
+
+    test_measurement_mean = computations.moving_mean(
+        test_measurement, epoch_length=EPOCH_LENGTH
+    )
+
+    assert np.allclose(test_measurement_mean.measurements, expected_mean)
+    assert test_measurement_mean.measurements.ndim == expected_mean.ndim
+    assert np.isclose(
+        test_measurement_mean.time.shape[0], (test_time.shape[0] / EPOCH_LENGTH)
```

Comment:
You could extract this into a variable inside the arrange e.g. `expected_time_shape = ...` so the assertion reads more simply.

Also, the shape array should contain integers, not floats, so use np.isequal in this case. Make sure to set expected_time_shape to an int (and to floor/ceil correctly) before asserting.